libvast/vast/detail/lru_cache.hpp

```diff
@@ -16,12 +16,16 @@
 // - Created:    June 20, 2013, 5:09 PM
 // - License:    BSD 3-Clause
 //
-// In addition to the cosmetic changes, the use of exceptions has been removed
-// from the original code and a `factory` member was added to create new entries
-// if a key is missing from the cache.
-// Additionally, iteration support and `resize()` and `clear()` function were
-// added; and `exists()` was renamed to `contains()` for closer alignment with
-// the standard library containers.
+// The following extensive changes have been applied to the original source:
+//  * Cosmetic refactoring to fit the VAST code style
+//  * Use of exceptions has been removed
+//  * A `factory` member was added to create new entries if a key is
+//    missing from the cache.
+//  * Iteration support
+//  * Member function `exists()` was renamed to `contains()`
+//  * Added member function `resize()`
+//  * Added member function `clear()`
+//  * Added member function `eject()`
```

Comment:
Do these changes need to be reflected in LICENSE.3rdparty?

Reply:
I don't think so, we're not adding any new third-party code but rather add our own modifications to existing code.